File: codeup/100000572/B.cpp
#include <iostream>
using namespace std;
int main() {
  struct student {
    int num;
    char name[20];
    char sex;
    int age;
  };

  student list[20];
  int n;
  scanf("%d", &n);
  for (int i = 0; i < n; i++) {
    scanf("%d %s %c %d", &list[i].num, list[i].name, &list[i].sex,
          &list[i].age);
  }
  student *temp;
  for (int i = 0; i < n; i++) {
    temp = list + i;
    printf("%d %s %c %d\n", temp->num, temp->name, temp->sex, temp->age);
  }
  return 0;
}

File: codeup/100000572/B.py
#!/usr/bin/python3
class student:
    def __init__(self, num, name, sex, age):
        self.num = num
        self.name = name
        self.sex = sex
        self.age = age


n = int(input())
student_list = []
for i in range(n):
    num, name, sex, age = list(input().split())
    student_list.append(student(num, name, sex, age))

for i in range(n):
    print(student_list[i].num, student_list[i].name,
          student_list[i].sex, student_list[i].age)
